src/richie/apps/core/templates/djangocms_video/default/video_player.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n cms_tags %}
```

Comment:
> Unfortunaly due to order of "INSTALLED_APPS" setting, this does
not work. "djangocms_video" is loaded before "richie.core" and so the first
win the loading match. However i altered locally my settings to test this
fix correctly fix the validation issue.

Then we should change the order in `INSTALLED_APPS` to have the correct order?

Reply:
This would resolve our issue and all other with plugins and external applications, yes.

But i'm not sure enough about consequences. From what i see at `src/richie/apps/core/templates/`, it should be safe but i request someone else to validate this.

It validated, from what i talked with samuel, the same change should be applied to project which embed richie like your "ongoing" funmooc.fr reboot with richie and so to fun coorporate, so their template directory should be applied on top of richie which should be on top of cms apps and plugins.

Reply:
@mbenadda @sveetch after reviewing this, it appears that the proposed change only affects the sandbox... it is thus something that has to be done in each project using Richie and not here. I propose to remove this commit and merge the rest now. WDYT?

Reply:
@sampaccoud There is still two issue to fix which are about homepage and course glimpses